flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/checkpoints/CheckpointingStatistics.java

```diff
@@ -390,6 +390,8 @@ public static final class RestoredCheckpointStatistics {
 
         public static final String FIELD_NAME_IS_SAVEPOINT = "is_savepoint";
 
+        public static final String FIELD_NAME_SAVEPOINT_FORMAT = "savepoint_format";
```

Comment:
Use camelCase for new fields

Reply:
I'll take this comment, but I haven't found documentation for such code style requirements. Could you please post a link to the relevant discussion? Thank you.

Reply:
FLINK-15787

Reply:
@zentol I think we can add this format request in [Flink's code style guide](https://flink.apache.org/contributing/code-style-and-quality-formatting.html). Not everyone knows the existence of FLINK-15787, not to mention this ticket has been inactive for more than 2 years.